### ISSUE TITLE
LOSS

### ISSUE THREAD

**Issue by AmberHen** [L1aoXingyu/fcn.pytorch#3]
Why does the training loss start directly from 0? If I don't use pre-trained weights, the loss starts at 0.01. Is this what is the reason? Thank you.
